### DISCUSSION TITLE
Cannot calculate the sum when using if condition

### DISCUSSION THREAD

**Discussion by edigonzales** [jxlsteam/jxls#330] [Q&A]
Hi

I have a template with an if condition (see screenshot). I'm not able to calculate the sum of the values of one column. The sum row is not rendered at all. Is there something wrong with my expressions? It works without the if condition.

![Screenshot 2024-07-29 at 09 55 56](https://github.com/user-attachments/assets/142869df-757b-4e02-ab6e-4d13c3cfe5ea)

**Comment by SoltauFintel** [comment]
Try this: jx:area lastCell="C4"
jx:each lastCell="C3"
SUM over C2:C3

**Comment by edigonzales** [comment]
Thanks. I got a sum right now but the sum respects only the rows < 1000.

![Screenshot 2024-07-30 at 07 36 04](https://github.com/user-attachments/assets/c0a54456-af8f-41eb-a724-82be198a3361)

![Screenshot 2024-07-30 at 07 37 15](https://github.com/user-attachments/assets/ed326bb5-81e8-4a6a-9c76-645f521f7fe5)
